CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+0.1.4 (2015-11-12)
```

Comment:
> Can we call this DEV and bump version in its own pull request?

What do you mean, @blaix ? Should I remove all version-related changes from this PR, and file a new PR for them later?